components/builder-protocol/src/originsrv.rs

```diff
@@ -37,10 +37,10 @@ impl Routable for OriginGet {
 }
 
 impl Routable for OriginCreate {
-    type H = InstaId;
+    type H = String;
 
     fn route_key(&self) -> Option<Self::H> {
-        Some(InstaId(self.get_owner_id()))
+        Some(self.get_name().to_string())
```

Comment:
That's right. The `sessionsrv` is where this would go, although the more I think about it the more I suspect it should be the `accountsrv`. Anything that's being mapped to an internal user id will need to live there.